chapter01/c01_hallway.rpy

```diff
@@ -1,9 +1,9 @@
 label c01_hallway:
 
     Hmm... I really shouldn't disobey Meadow... I would probably be a nuisance for the people downstairs as well.
-    And besides, knowing Meadow, she'll tell me at least the basic story of what happened.
+    And besides, knowing Meadow, she'd tell me at least the basic story of what happened.
 
-    The time was now eight thirty, and Meadow still hasn't returned. [Pause] Something terrible really must have happened.
+    The time was now 8:30, and Meadow still had't returned. [Pause] Something terrible really must have happened.
```

Comment:
Subjective, but there are a lot of times around (like "seven twenty-five") that are very awkward to write as full words, so to standardize I've changed everything to use digits.